applications/rag/metadata.yaml

```diff
@@ -17,6 +17,12 @@ spec:
   content: {}
   interfaces:
     variables:
+      - name: additional_labels
```

Comment:
I thought @andrewsykim added this already

Reply:
It wasn't in alphabetical order when I added it, sorry!

Reply:
yeah it was reshuffled to alphabetical order